build.gradle

```diff
@@ -11,8 +11,18 @@ buildscript {
     ext {
         opensearch_version = System.getProperty("opensearch.version", "2.0.0-SNAPSHOT")
         knn_bwc_version = System.getProperty("bwc.version", "1.2.0.0-SNAPSHOT")
+        version_qualifier = System.getProperty("build.version_qualifier", "")
```

Comment:
Could you add a comment for to this and also provide link in comment to where OpenSearch discusses this scheme?

Reply:
Ack, will add a comment. Requirements from OpneSearch are listed in the issue itself - https://github.com/opensearch-project/k-NN/issues/315. They have started to produce build with this scheme - https://aws.oss.sonatype.org/content/repositories/snapshots/org/opensearch/opensearch/2.0.0-alpha1-SNAPSHOT/. Actually at the moment it's the only way to get 2.0.0 snapshot with lucene 9.1